source/tests/array_api_strict/descriptor/repformers.py

```
@@ -0,0 +1,98 @@
+                value = Atten2Map.deserialize(value.serialize())
+        elif name in {"attn2_mh_apply"}:
+            if value is not None:
+                value = Atten2MultiHeadApply.deserialize(value.serialize())
+        elif name in {"attn2_lm"}:
+            if value is not None:
+                value = LayerNorm.deserialize(value.serialize())
+        elif name in {"attn2_ev_apply"}:
+            if value is not None:
+                value = Atten2EquiVarApply.deserialize(value.serialize())
+        elif name in {"loc_attn"}:
+            if value is not None:
```

Ruff

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)